.github/workflows/build-appimage.yml

```diff
@@ -0,0 +1,24 @@
+name: build appimage
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: "**"
+ 
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
an app image should be using the oldest, not the youngest (i.e. latest) - or is latest here the oldest?

Reply:
this is just the image that runs the docker container that generates the appimage

Reply:
ah, ok